src/modules/gfx_maze.c

```diff
@@ -0,0 +1,234 @@
+// Simple maze animation.
+// Maze algorithm taken from: https://github.com/Figglewatts/mazegen
+// Thank you kind stranger!
+//
+// Copyright (c) 2019, Cyrill "xermic" Leutwiler <me@09f9.org>
```

Comment:
You might want to add attribution to the original author of the algorithm.

Reply:
will do